### ISSUE TITLE
Annotations do not appear when the time picker range is included in annotation start/end range

### ISSUE THREAD

**Issue by tpispirigkos** [grafana/grafana#18852]
**What happened**:
I have created an annotation with starting/ending time 12:00:00 - 12:30:00 .
When i chose in the time picker 10:00:00 - 13:00:00 the annotations was visible. When i changed the time picker to 12:10:00-12:20:00 the annotation did not appear in the graph.

**What you expected to happen**:
The annotation to appear in the graph.

**How to reproduce it (as minimally and precisely as possible)**:
Create an annotation and select as time picker so as annotation range includes time picker range. 

**Anything else we need to know?**:
-

**Environment**:
- Grafana version: 6.3.3
- Data source type & version: default/ random walk
- OS Grafana is installed on: docker
- User OS & Browser: chrome
- Grafana plugins: -
- Others: -

**Comment by torkelo** [comment]
The the range filter is exclusive.  @ryantxu maybe we should change this to be inclusive? `>= <=`

**Comment by ryantxu** [comment]
Hymm -- it looks like it is inclusive:
```
	if query.From > 0 && query.To > 0 {
		sql.WriteString(` AND annotation.epoch <= ? AND annotation.epoch_end >= ?`)
		params = append(params, query.To, query.From)
	}
```
https://github.com/grafana/grafana/blob/master/pkg/services/sqlstore/annotation.go#L179

**Comment by ryantxu** [comment]
wait -- reading the description... this should be fixed in master (6.4) the behavior you describe is what I would expect in 6.3.x

please re-open if you still have issues when running on master or 6.4
